src/components/CoinOverview/CoinOverview.js

```diff
@@ -118,23 +118,23 @@ class CoinOverview extends React.Component {
                   <td>
                     <DoubleSpan height="8px" width="100%" background="#2172e5">
                       <Styled24VMarketCapTextWrapper
-                        width={(obj.total_volume / obj.market_cap) * 100}
+                        width={(coin.total_volume / coin.market_cap) * 100}
                         background={"white"}
                       >
                         <p>
                           $
                           {Intl.NumberFormat("en-US", {
                             notation: "compact",
                             maximumFractionDigits: 1,
-                          }).format(obj.circulating_supply)}
+                          }).format(coin.circulating_supply)}
                         </p>
 
                         <p>
                           $
                           {Intl.NumberFormat("en-US", {
                             notation: "compact",
                             maximumFractionDigits: 1,
-                          }).format(obj.total_supply)}
+                          }).format(coin.total_supply)}
```

Comment:
this int function is being reused, create a separate formatter function

Reply:
done with numerals.js